app/serializers/api/v1/participant_from_induction_record_serializer.rb

```diff
@@ -60,12 +60,12 @@ def eligible_for_funding?(profile)
 
       attribute :mentor_id do |induction_record|
         if induction_record.participant_profile.ect?
-          induction_record.participant_profile.mentor&.id
+          induction_record.mentor_profile&.user&.id
```

Comment:
Think this might need to be this?
```suggestion
          induction_record.mentor_profile&.participant_identity&.external_identifier
```